examples/practice-typescript/src/scripts/app.tsx

```diff
@@ -0,0 +1,37 @@
+import Logo from 'components/Logo/Logo';
+import SideBarMenu from 'components/SideBarMenu/SideBarMenu';
+import SideBarMenuItem, {
+  SideBarMenuItemProps,
+} from 'components/SideBarMenuItem/SideBarMenuItem';
+
+const MENU: SideBarMenuItemProps[] = [
+  {
+    id: '1',
+    text: 'User management',
+    router: '/component.html',
+    isActive: true,
+  },
+  {
+    id: '2',
+    text: 'Product management',
+    router: '/component.html',
+    isActive: false,
+  },
+];
+
+const App = () => {
+  return (
```

Comment:
is you have list menu items, you could get first item then fill it data here instead of hard-core the value.